flutter/lib/src/event_processor/screenshot_event_processor.dart

```diff
@@ -1,20 +1,27 @@
 import 'dart:async';
-import 'dart:math';
 import 'dart:typed_data';
 import 'dart:ui';
 
-import 'package:flutter/rendering.dart';
 import 'package:meta/meta.dart';
 import 'package:sentry/sentry.dart';
+import '../screenshot/recorder.dart';
+import '../screenshot/recorder_config.dart';
 import '../screenshot/sentry_screenshot_widget.dart';
 import '../sentry_flutter_options.dart';
-import '../renderer/renderer.dart';
 import 'package:flutter/widgets.dart' as widget;
 
 class ScreenshotEventProcessor implements EventProcessor {
   final SentryFlutterOptions _options;
 
-  ScreenshotEventProcessor(this._options);
+  late final ScreenshotRecorder _recorder;
+
+  ScreenshotEventProcessor(this._options) {
+    _recorder = ScreenshotRecorder(
+      ScreenshotRecorderConfig(quality: _options.screenshot.screenshotQuality),
```

Comment:
Instead of passing quality here, we should pass the width & height that corresponds to the configured quality.

Something like this:

```suggestion
      final size = _options.screenshot.quality.targetResolution();
      _recorder = ScreenshotRecorder(
        ScreenshotRecorderConfig(width: size, height: size),
```